<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="1" failed="0" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[login method is executing]]>
    </line>
  </reporter-output>
  <suite started-at="2022-04-30T13:45:25 IST" name="Suite" finished-at="2022-04-30T13:46:27 IST" duration-ms="62307">
    <groups>
    </groups>
    <test started-at="2022-04-30T13:45:25 IST" name="Test" finished-at="2022-04-30T13:46:27 IST" duration-ms="62307">
      <class name="test.ActitimeLoginLogout">
        <test-method is-config="true" signature="setup()[pri:0, instance:test.ActitimeLoginLogout@548d708a]" started-at="2022-04-30T13:45:25 IST" name="setup" finished-at="2022-04-30T13:46:07 IST" duration-ms="42304" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="logintoapp()[pri:0, instance:test.ActitimeLoginLogout@548d708a]" started-at="2022-04-30T13:46:07 IST" name="logintoapp" finished-at="2022-04-30T13:46:27 IST" duration-ms="19444" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logintoapp -->
        <test-method signature="login()[pri:0, instance:test.ActitimeLoginLogout@548d708a]" started-at="2022-04-30T13:46:27 IST" name="login" finished-at="2022-04-30T13:46:27 IST" duration-ms="31" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[login method is executing]]>
            </line>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method is-config="true" signature="logoutfromapp()[pri:0, instance:test.ActitimeLoginLogout@548d708a]" started-at="2022-04-30T13:46:27 IST" name="logoutfromapp" finished-at="2022-04-30T13:46:27 IST" duration-ms="379" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "Object.getClass()" because "arg" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "Object.getClass()" because "arg" is null
at org.openqa.selenium.remote.internal.WebElementToJsonConverter.apply(WebElementToJsonConverter.java:60)
at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948)
at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913)
at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578)
at org.openqa.selenium.remote.RemoteWebDriver.executeScript(RemoteWebDriver.java:477)
at pom.ActitimeHomePage.logoutMethod(ActitimeHomePage.java:101)
at generics.BaseTest.logoutfromapp(BaseTest.java:41)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700)
at org.testng.internal.TestInvoker.runAfterGroupsConfigurations(TestInvoker.java:676)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:660)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutfromapp -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:test.ActitimeLoginLogout@548d708a]" started-at="2022-04-30T13:46:27 IST" name="teardown" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1651306587650" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- test.ActitimeLoginLogout -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
